doc/source/Getting_started/Installation.rst

```diff
@@ -9,23 +9,30 @@ basic support for EDB and Circuit (Nexxim).
 Requirements
 ~~~~~~~~~~~~
 In addition to the runtime dependencies listed in the installation information, PyAEDT
-requires AEDT 2021 R1 or later.
+requires Ansys Electronics Desktop (AEDT) 2021 R2 or later. AEDT Student Version is also supported.
```

Comment:
```suggestion
requires Ansys Electronics Desktop (AEDT) 2021 R2 or later. The AEDT Student Version is also supported.
```
Is there a reason that you think AEDT should be spelled out here? Is this the first occurrence if a reader starts at the beginning of the doc?